bridges/InstagramBridge.php

```diff
@@ -50,6 +51,38 @@ class InstagramBridge extends BridgeAbstract {
 	const TAG_QUERY_HASH = '9b498c08113f1e09617a1703c22b2f32';
 	const SHORTCODE_QUERY_HASH = '865589822932d1b43dfe312121dd353a';
 
+
+	const BIBLIOGRAM_ACCEPTED = false;
+	/* 	Video/image from Instagram now can't display due to SameOrigin policy.
+	*		This flag will allow to use Bibliogram to process image/video from Instagram
+	*		Value : {true|false}
+	*		Default: false
+	*/
+	const BIBLIOGRAM_URL = 'https://bibliogram.art/';
+	/* You can change to your perferred one.
+	*	List of instance: https://git.sr.ht/~cadence/bibliogram-docs/tree/master/docs/Instances.md
+	*/
```

Comment:
@dhuschde set the flag on line 55 to true and have your "Use direct media links" ticked as well.

If you want to change instance, you can replace url on line 61. This will only supports Bibliogram.